cla_backend/settings/base.py

```diff
@@ -333,10 +336,22 @@
 
 BROKER_TRANSPORT_OPTIONS = {
     "polling_interval": 10,
-    "region": "eu-west-1",
+    "region": os.environ.get("SQS_REGION", "eu-west-1"),
     "wait_time_seconds": 20,
     "queue_name_prefix": _queue_prefix % {"env": CLA_ENV},
 }
+
+if "CELERY_PREDEFINED_QUEUE_URL" in os.environ:
```

Comment:
I think I would prefer this to check the truthiness of that value, rather than just the existence of the key. I'm just a bit concerned that a value of `None` would break things in a way that you wouldn't expect without reading and understanding this bit of code.

Reply:
updated with `if os.environ.get("CELERY_PREDEFINED_QUEUE_URL"):`